File: generators/java/src/main/java/dqualizer/dqlang/archive/k6configurationrunner/dqlang/options/Stage.java
package dqualizer.dqlang.archive.k6configurationrunner.dqlang.options;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class Stage {

    private String duration;
    private int target;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/archive/k6configurationrunner/dqlang/options/Stage.kt
package io.github.dqualizer.dqlang.archive.k6configurationrunner.dqlang.options

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.ToString

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
class Stage {
    val duration: String? = null
    val target = 0
}
